src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php

```diff
@@ -118,6 +118,13 @@ public function dump(Data $data)
             $name = substr($name, strrpos($name, '/') + 1);
         }
 
+        if ('cli-server' === PHP_SAPI) {
+            $cloner = new VarCloner();
+            $dumper = new CliDumper('php://stderr', $this->charset);
+            $dumper->dump($cloner->cloneVar(sprintf('In %s on line %d', $file, $line)));
```

Comment:
I think so, but maybe in a different color/background. That's the kind of things we can tweak if the main idea is appeal to a majority.